administration/src/bp-modules/applications/ApplicationsOverview.tsx

```diff
@@ -76,6 +76,9 @@ const ApplicationsOverview = (props: { applications: Application[] }) => {
                 key={application.id}
                 application={application}
                 gotDeleted={() => setUpdatedApplications(sortedApplications.filter(a => a !== application))}
+                gotChanged={application =>
```

Comment:
"gotChanged" seems to be the name of a boolean. As this is a function, i would expect it to be named something like "onChange".
(Maybe you can also adjust the "gotDeleted" naming)